source/Ship.h

```diff
@@ -170,6 +170,10 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	void SetPosition(Point position);
 	// When creating a new ship, you must set the following:
 	void Place(Point position = Point(), Point velocity = Point(), Angle angle = Angle(), bool isDeparting = true);
+    double turretRange = 0;
+    double gunRange = 0;
```

Comment:
these should not be here, they should be private

Reply:
Why? I understand that it is best practices for data security to have the variable be private and then have encapsulation to extricate the info inside the pirvate variable; however in practice that just seem to add extra steps for no obvious benefit.

That being said, I'm trying to change this, and having trouble getting it to successfully build. Moving those variables into a private area is easy enough, but I am failing to figure out how to encapsulate it properly.

Beyond that, this looks like the same sort of situation as the antiMissileRange value, which is likewise a public variable.